Resources/Maps/_Mono/Shuttles/framework.yml

```
@@ -0,0 +1,541 @@
+          - 0
+          - 0
+          - 0
+          - 0
+        chunkSize: 4
+    - type: GasTileOverlay
+    - type: RadiationGridResistance
+    - type: BecomesStation
+      id: UNJ
+- proto: AirlockGlass
+  entities:
+  - uid: 20
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)